apps/gallery/js/ImageEditor.js

```diff
@@ -952,27 +927,67 @@ ImageEditor.prototype.destroy = function() {
 // desired edits change or when the size of the container changes (on
 // orientation change events, for example)
 ImageEditor.prototype.edit = function(callback) {
-  if (!this.preview.src) {
+  if (!callback && this.editing) {
+    this.needsAnotherEdit = true;
+    return;
+  }
+  this.editing = true;
+
```

Comment:
It will help if we can add comment that explains this.editing and this.needsAnotherEdit